app/routes/goals_routes.py

```diff
@@ -0,0 +1,115 @@
+from flask import Blueprint, jsonify, abort, make_response, request
+from app.models.goal import Goal
+from app.models.task import Task
+from app.routes.helper_funcs import get_valid_item_by_id
+from app import db
+
+# All routes for goals start with "/goals" URL prefix
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+
+@goals_bp.route("", methods=['POST'])
+def create_goal():
+    request_body = request.get_json()
+
+    if "title" not in request_body:
+        return {
+            "details": "Invalid data"
+        }, 400
+    new_goal = Goal.from_dict(request_body)
+
+    db.session.add(new_goal)
+    db.session.commit()
```

Comment:
Great job using `db.session.<method-name>.` Session gives us access to the following:

* `db` is our app’s instance of SQLAlchemy.
* `session` represents our active database connection.
* By referencing db.session we can use SQLAlchemy’s methods to perform tasks like committing a change to a model, storing a new record of a model, and deleting a record.

By referencing `db.session.add()` you are able to use the SQLAlchemy method to store a new record of the Goal model. ⭐